static/app/views/settings/account/notifications/utils.tsx

```diff
@@ -436,3 +439,11 @@ export function getDocsLinkForEventType(event: 'error' | 'transaction' | 'attach
       return 'https://docs.sentry.io/product/accounts/quotas/manage-event-stream-guide/#common-workflows-for-managing-your-event-stream';
   }
 }
+
+export function getNotificationTypeFromPathname(name: string) {
+  const result = Object.entries(NOTIFICATION_SETTINGS_PATHNAMES).find(
+    ([_, pathname]) => pathname === name
+  );
+
+  return result ? result[0] : '';
```

Comment:
Instead of falling back to `''`, we should probably fall back to the string that was passed in. I'd also rename it from `name` to something more descriptive.

E.g.
```js
export function getNotificationTypeFromPathname(routerPathname: string) {
  const result = Object.entries(NOTIFICATION_SETTINGS_PATHNAMES).find(
    ([_, pathname]) => pathname === routerPathname
  ) ?? [routerPathname];
  return result[0];
}
```

Bonus points if you add a quick docstring :)